src/main/java/net/gecore/streamerendpoints/service/igdb/IGDBGameService.java

```diff
@@ -0,0 +1,41 @@
+package net.gecore.streamerendpoints.service.igdb;
+
+import net.gecore.streamerendpoints.configuration.TwitchConfig;
+import net.gecore.streamerendpoints.service.twitch.AuthService;
+import net.gecore.streamerendpoints.service.twitch.TwitchAPIException;
+
+import net.gecore.streamerendpoints.service.utils.JsonPathUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class IGDBGameService {
+    private final IGDBAPI igdbapi;
+    private final TwitchConfig twitchConfig;
+    private final AuthService authService;
+//    private final Logger LOGGER = LoggerFactory.getLogger(IGDBService.class);
+
+    public IGDBGameService(IGDBAPI igdbapi, TwitchConfig twitchConfig, AuthService authService){
+        this.igdbapi = igdbapi;
+        this.twitchConfig = twitchConfig;
+        this.authService = authService;
+    }
+
+    public String retrieveGameById(Integer gameId)
+            throws IGDBAPIException, TwitchAPIException, MalformedURLException {
+        URL url = new URL("https://api.igdb.com/v4/games");
```

Comment:
I'd rather already see in this pull request that the url buildup is similar to the ones used in the twitch specific services. For now simply turning the last part of the URL, `games` and `external games`, into values of an 'IGDBEndpoint' enum should work.

Reply:
Would you rather I also make a URLHelper class in the `igdb` package, or better yet add a `buildIGDBUrl` method to `URLHelper` and move it to the `utils` package?

Reply:
Yeah I was going to do the latter afterwards, but you can do it too. Make sure to rename the old one to buildTwitchUrl.